### ISSUE TITLE
Word Cloud Returns No Data

### ISSUE THREAD

**Issue by sunsan01** [hpcc-systems/Visualization#2125]
It appears that Word Cloud is not working when selected.

**Comment by GordonSmith** [comment]
What version - it appears to be working in the following version (if you switch the Pie to the word cloud):  
* http://rawgit.com/hpcc-systems/Visualization/master/demos/dermatology.html?src/marshaller/HTML.databomb
* http://rawgit.com/hpcc-systems/Visualization/v1.14.8-rc2/demos/dermatology.html?src/marshaller/HTML.databomb
* http://rawgit.com/hpcc-systems/Visualization/v1.14.6/demos/dermatology.html?src/marshaller/HTML.databomb

**Comment by sunsan01** [comment]
Not sure why it's not working in my dashboard. is there any limitations on using word cloud? maybe my input data is too large?

**Comment by sunsan01** [comment]
Ok. I figured out what the problem is now. I think it has to do with the fact that I wanted a value under Measures to be displayed, rather than attributes, so word cloud stopped working when that is applied. Can we make that an option? also, when I tried creating an interaction between the Word Cloud widget and a table, the interaction appears to not be working. I can't select one single element within the word cloud to be displayed in the table. thanks!
